projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -48,6 +48,7 @@ ADDITIONAL_PAUSE_1-22_FROM=1-21
 ADDITIONAL_PAUSE_1-23_FROM=1-22
 ADDITIONAL_PAUSE_1-24_FROM=1-23
 ADDITIONAL_PAUSE_1-25_FROM=1-24
+ADDITIONAL_PAUSE_1-26_FROM=1-26
```

Comment:
Pause image tags match in links below for 1.26, so this does not need to reference the previous tag.

1. https://github.com/kubernetes/kubernetes/blob/v1.26.0/cmd/kubeadm/app/constants/constants.go#L420
2. https://github.com/kubernetes/kubernetes/blob/v1.26.0/build/pause/Makefile#L20

So I wasn't going to add this line, but the image builder presubmit fails without it

Reply:
that makes sense. Id just leave it